code/lib/addons/package.json

```diff
@@ -31,7 +39,17 @@
   ],
   "scripts": {
     "check": "tsc --noEmit",
-    "prepare": "node ../../../scripts/prepare.js"
+    "prepare": "../../../scripts/prepare/bundle.ts"
+  },
+  "bundler": {
+    "entries": [
+      "./src/index.ts",
+      "./src/hooks.ts",
+      "./src/main.ts",
+      "./src/make-decorator.ts",
+      "./src/storybook-channel-mock.ts",
+      "./src/types.ts"
```

Comment:
I have the feeling, that only `./src/index.ts` is needed here.

Reply:
Solved.